bridge-token-factory/src/lib.rs

```diff
@@ -360,33 +400,57 @@ impl BridgeTokenFactory {
 
         let required_deposit = self.record_proof(&proof);
 
-        assert!(
-            env::attached_deposit()
-                >= required_deposit + self.bridge_token_storage_deposit_required
+        let Recipient { target, message } = parse_recipient(new_owner_id);
+
+        let token = EthAddressHex(token);
+        let amount_to_transfer: u128;
+        let fee_amount = self.calculate_deposit_fee_amount(
+            &token,
+            amount,
+            message.as_ref().and_then(|_| Some(&target)),
         );
+        amount_to_transfer = amount - fee_amount;
 
-        let Recipient { target, message } = parse_recipient(new_owner_id);
+        let multiplier = if fee_amount > 0 { 1 } else { 2 };
```

Comment:
If we transfer to aurora the multiplier should be 1 anyway, but we should add 1 yocto token

Reply:
Added 1 yocto, but the new storage deposit logic was reverted, so all storage deposit calls that related to the fee were removed. a6395e936c38f6adfcbb895c29c2b5cb5766b591.
The storage deposit is done for the factory account on deploying the new token. And storage deposits should be done manually for already exits tokens manually.